README.md

```diff
@@ -17,7 +17,6 @@ and [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-
 exceptions/differences/extensions (:white_check_mark: are the implemented sniffs):
 
 - Keep the nesting of control structures per method as small as possible
-- Prefer early exit over nesting conditions or using else
```

Comment:
This wording is maybe a little bit more clear than the one a few line down. What do you think ?

Reply:
@mikeSimonson Good idea, gonna use it :)